lib/components/atoms/Button/tests/__snapshots__/Button.style.test.js.snap

```diff
@@ -59,6 +59,8 @@ exports[`Button should render correctly 1`] = `
         "CTA_TERTIARY": "
     border: var(--border-size-cta-tertiary) solid var(--border-secondary);
   ",
+        "DD_FILE_FIELD": "border: var( --border-size-field-active) dashed var(--border-primary);",
```

Comment:
These names are not declarative enough, I don't know what DD and FF mean

Reply:
These names have been updated respective to component which they are used in.